frontend/src/App.tsx

```diff
@@ -39,16 +40,24 @@ const App = () => {
           </Snackbar>
           <CssBaseline />
           <Header authSession={authSession} />
-          <Routes>
-            <Route path='/' element={<LandingPage authSession={authSession} />} />
-            <Route path='/Elections' element={<Elections authSession={authSession} />} />
-            <Route path='/Login' element={<Login />} />
-            <Route path='/Debug' element={<DebugPage authSession={authSession} />} />
-            <Route path='/CreateElection' element={<AddElection authSession={authSession} />} />
-            <Route path='/Election/:id/*' element={<Election authSession={authSession} />} />
-            <Route path='/DuplicateElection/:id' element={<DuplicateElection authSession={authSession} />} />
-            <Route path='/Sandbox' element={<Sandbox />} />
-          </Routes>
+          <Box
+            sx={{
+              width: '100%',
+              minHeight: `calc(100vh - 255px - 64px)` //Sets min height of content to 100% minus footer and header in order to force footer to the bottom
```

Comment:
This height logic will probably break with the new responsive header that can take multiple rows

Maybe you can put the header, middle, and footer all in a flex box and set the middle to flex grow?

Reply:
Done, but by setting footer margin top to auto.